requirements.yml

```diff
@@ -1,3 +1,8 @@
+- src: geerlingguy.php-mysql
```

Comment:
@seth-shaw-unlv should this have a version to pin it to?

Reply:
We can. I will update the PR to pin the current version.